README.md

```diff
@@ -156,6 +156,7 @@ App|Description
 [uart_tx](pio/uart_tx)| Implement the transmit component of a UART serial port, and print hello world.
 [ws2812](pio/ws2812)| Examples of driving WS2812 addressable RGB LEDs.
 [addition](pio/addition)| Add two integers together using PIO. Only around 8 billion times slower than Cortex-M0+.
+[pio_isr](pio/isr)| Respond to interrupts generated by a PIO state machine.
```

Comment:
For consistency with the other examples listed here, should this be `[isr](pio/isr)` ? Sorry for being picky!

Reply:
No problem, I understand! I used pio_isr because it's the name of the program that gets compiled, but I can change it to isr if you'd like.

Reply:
Updated, but also renamed.